### PR TITLE
define $PYTHONNOUSERSITE in PythonPackage easyblock

### PR DESCRIPTION
**Pull request by boegel**
this shouldn't break anything (unless it's already broken)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyblocks/generic/pythonpackage.py [hunk 1]
```diff
@@ -148,6 +148,11 @@ def configure_step(self):
         run_cmd("which python")
         run_cmd("python -c 'import sys; print(sys.executable)'")
 
+        # don't add user site directory to sys.path (equivalent to python -s)
+        # see https://www.python.org/dev/peps/pep-0370/
+        env.setvar('PYTHONNOUSERSITE', '1')
+        run_cmd("python -c 'import sys; print(sys.path)'")
```
Comment (wpoely86) [inline easybuild/easyblocks/generic/pythonpackage.py]: why the run?

Comment (boegel) [inline easybuild/easyblocks/generic/pythonpackage.py]: to collect debug info

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Pull request merged**